pkg/ebpf/lockcontention.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	// maximum lock ranges to track
 	maxTrackedRanges = 16384
+
+	// batch size when update per cpu map storing lock ranges
+	updateBatchSize = 100
```

Comment:
can you elaborate the 100? is this the number of entries? what is the expected size of each entry?

[example](https://github.com/DataDog/datadog-agent/blob/main/pkg/network/protocols/events/configuration.go#L32-L35) for comment